gems/pending/util/miq-password.rb

```diff
@@ -123,7 +123,23 @@ def self.clear_keys
   end
 
   def self.v2_key
-    @v2_key ||= ez_load("#{key_root}/v2_key")
+    @v2_key ||= begin
+      key_file = File.expand_path("v2_key", key_root)
+      if File.exist?(key_file)
+        ez_load(key_file)
+      else
+        msg = <<-EOS
+#{key_file} doesn't exist!
+On an appliance, it should be generated on boot by evmserverd.
```

Comment:
Still don't like this here.  I feel like MiqPassword is one of those classes that is pure, and it feels weird referencing things it doesn't/shouldn't know about.  But, I can merge for now and we can clean it up later.